### DISCUSSION TITLE
Best solution for distributed build

### DISCUSSION THREAD

**Discussion by cyb3rslack3r** [Security-Onion-Solutions/securityonion#5306] [Unsupported Versions] [answered]
I have 2 servers (24 cores, 256GB RAM and 24 TBs SSD) I am using to build a distributed build to connect to a 10gb link.  I wanted to build one as a sensor in its own server to prioritize capturing at high speed, but not sure the best configuration for the other server.  Trying the Manager/Search configuration on bare bones, but wondered if it would be preferred to run ESXi and have one VM as the Manager and one or two more VMs running as search agents.  Basically wondering what the recommended config for high speed collection with only 2 servers.  Thanks

**Comment by weslambert** [comment, marked as answer]
If you have no other infrastructure, then it might actually be best to separate the manager/search components on top of the hypervisor for one box. There is a fair bit of "it depends". Unfortunately, we can't really scope your deployment within this forum, but hopefully that helps.
